sdks/go/pkg/beam/registration/registration.go

```diff
@@ -0,0 +1,6990 @@
+// File generated by specialize. Do not edit.
```

Comment:
I'm just excited that the new file/diff is now small enough to show up in the diff UI. :D